data-prepper-plugins/csv-processor/src/test/java/org/opensearch/dataprepper/plugins/codec/csv/CsvCodecTest.java

```diff
@@ -282,7 +280,7 @@ void test_when_manualHeaderTooManyColumns_then_omitsExtraColumnsOnParsedEvents(f
 
     @ParameterizedTest
     @ValueSource(ints = {2, 10, 100})
-    void test_when_autoDetectHeaderWrongNumberColumnsAndJaggedRows_then_skipsRemainingRows(final int numberOfRows) throws IOException {
```

Comment:
Similar to my comment above about https://github.com/opensearch-project/data-prepper/commit/bc75494167d6befe3778493749678a501afbe1a9, this PR currently is reverting changes in main that need to remain.

You should revert back the changes starting at this line. There should be no diffs from this point forward.

Also, please check the imports.

Reply:
Done! Thanks for pointing out.